lib/ansible/modules/shell.py

```diff
@@ -73,10 +73,10 @@
     version_added: "2.8"
 notes:
   - If you want to execute a command securely and predictably, it may be
-    better to use the M(command) module instead. Best practices when writing
-    playbooks will follow the trend of using M(command) unless the C(shell)
-    module is explicitly required. When running ad-hoc commands, use your best
-    judgement.
+    better to use the M(ansible.builtin.command) module instead. Best practices
+    when writing playbooks will follow the trend of using M(ansible.builtin.command)
+    unless the C(shell) module is explicitly required. When running ad-hoc
```

Comment:
```suggestion
    unless the M(ansible.builtin.shell) module is explicitly required. When running ad-hoc
```

Reply:
I was not sure about this one, I thought it was a configuration option as the `C()` implies

Reply:
unless the C(shell) `module` is explicitly required. I think the module is implied from the context

Reply:
not only because of the word `module`, from written a bit above